From 7ed97594998bcdce10bbcb50068335cd1a072807 Mon Sep 17 00:00:00 2001
From: Drew Fustini <drew@beagleboard.org>
Date: Sun, 30 May 2021 15:41:37 -0700
Subject: [PATCH 51/56] riscv: dts: starfive-jh7100: add
 SPDX-License-Identifier

Add SPDX-License-Identifier and copyright information

Signed-off-by: Drew Fustini <drew@beagleboard.org>
---
 arch/riscv/boot/dts/starfive/jh7100-starlight.dts | 3 +++
 arch/riscv/boot/dts/starfive/jh7100.dtsi          | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/arch/riscv/boot/dts/starfive/jh7100-starlight.dts b/arch/riscv/boot/dts/starfive/jh7100-starlight.dts
index 6b18845345e3..1685f676815d 100644
--- a/arch/riscv/boot/dts/starfive/jh7100-starlight.dts
+++ b/arch/riscv/boot/dts/starfive/jh7100-starlight.dts
@@ -1,3 +1,6 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+/* Copyright (c) 2021 StarFive Technology Co., Ltd. */
+
 /dts-v1/;
 #include "jh7100.dtsi"
 #include <dt-bindings/starfive_fb.h>
diff --git a/arch/riscv/boot/dts/starfive/jh7100.dtsi b/arch/riscv/boot/dts/starfive/jh7100.dtsi
index 413455b8d14f..debbe0e4e4eb 100644
--- a/arch/riscv/boot/dts/starfive/jh7100.dtsi
+++ b/arch/riscv/boot/dts/starfive/jh7100.dtsi
@@ -1,3 +1,6 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
+/* Copyright (c) 2021 StarFive Technology Co., Ltd. */
+
 /dts-v1/;
 #include <dt-bindings/starfive_fb.h>
 #include <dt-bindings/gpio/gpio.h>
-- 
2.30.2

